### ISSUE TITLE
[feature request] Adding Padding functions to nn.functional

### ISSUE THREAD

**Issue by rasbt** [pytorch/pytorch#5083]
I noticed that the padding layers are missing in `torch.nn.functional`. For consistency, e.g., similar to the pooling ops etc., I think it would be a good idea to add these. 

If that sounds like a good idea, I am happy to help with this via a PR.

**Comment by soumith** [comment]
@rasbt have you looked at torch.nn.functional.pad ?
http://pytorch.org/docs/0.3.0/nn.html?highlight=pad#torch.nn.functional.pad

**Comment by rasbt** [comment]
Ah, thanks! I was looking for "padding" in the docs (similar to the organization in torch.nn) and just see now that the `pad` function is under "Vision Functions". That would solve the problem, sorry about that! 
![screen shot 2018-02-06 at 1 42 59 pm](https://user-images.githubusercontent.com/5618407/35877620-dad467ec-0b43-11e8-8790-95df01613586.png)
